validation-test/compiler_crashers_2_fixed/28872-rdar45583135.swift

```diff
@@ -0,0 +1,16 @@
+// This source file is part of the Swift.org open source project
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+
+// RUN: not %target-swift-frontend %s -emit-sil
```

Comment:
Once we fix the real issue here, this code will actually type check (and the test can fail, so we can update it then)